### DISCUSSION TITLE
Documentation and standardization for CLI

### DISCUSSION THREAD

**Discussion by jasonkarns** [naugtur/can-i-ignore-scripts#15] [Ideas]
Without Issues enabled, I assume discussions is where we're supposed to report issues?

`can-i-ignore-scripts --help` and `can-i-ignore-scripts -h` runs the CLI instead of printing any help.

I would like to see a more unix friendly CLI. Should be able to get help output. Running with unknown/unrecognized arguments should exit with an error and print help. 

Feature requests for the output:
Verbosity and output could be customized.
- be able to hide the "can i ignore" banner.  The banner is also not critical output so it should probably be printed to STDERR, not STDOUT. 
- Make the core output parseable (as a CLI, I would expect TSV output so that it's parseable and pipeable to other utilities).
- Be able to hide reporting of "safe to ignore" packages so I only see packages that require action.

**Comment by naugtur** [comment]
Issues are off to encourage PRs
If you want the features in happy to give you the intro to the code and review what you make. 

All 3 feature ideas make sense. When you get to submitting your contribution make sure each feature is a separate PR.
